src/staging/staging_main.py

```diff
@@ -249,37 +248,26 @@ def run_staging(  # noqa: C901
             rd_read_csv,
         )
 
-        # seaparate PNP data from full_responses (BERD data)
+        # filter for either PNP data or BERD data
         if stage_frozen_snapshot or stage_updated_snapshot:
-            full_responses, pnp_full_responses = helpers.filter_pnp_data(full_responses)
+            full_responses = helpers.filter_pnp_data(full_responses, config)
 
         if config["global"]["output_full_responses"]:
-            StagingMainLogger.info("Starting output of staged BERD data...")
+            survey_type = config["survey"]["survey_type"]
+            StagingMainLogger.info(f"Starting output of staged {survey_type} data...")
             staging_folder = staging_dict["staging_output_path"]
             tdate = datetime.now().strftime("%y-%m-%d")
             survey_year = config["survey"]["survey_year"]
             staged_filename = (
```

Comment:
I suppose you can use your new function for this now

Reply:
I tried to preserve the original name format as much as possible.